tests/bunit.core.tests/TestServiceProviderTest.cs

```diff
@@ -122,5 +130,24 @@ public void Test020()
 
 			actual.ShouldBe(expected);
 		}
+
+		[Fact(DisplayName = "No registered service returns null")]
+		public void Test021()
+		{
+			using var sut = new TestServiceProvider();
+			var result = sut.GetService<DummyService>();
+
+			Assert.Null(result);
+		}
```

Comment:
```suggestion
		[Fact(DisplayName = "No registered service returns null")]
		public void Test021()
		{
			using var sut = new TestServiceProvider();
			
			var result = sut.GetService<DummyService>();

			Assert.Null(result);
		}
```

I like to have an implicit "Arrange-Act-Assert"  sectioning in my tests, defined by spaces.